improver/categorical/utilities.py

```diff
@@ -576,3 +576,22 @@ def day_night_map(decision_tree: Dict[str, Dict[str, Union[str, List]]]) -> Dict
         for k, v in decision_tree.items()
         if "if_night" in v.keys()
     }
+
+
+def dry_map(decision_tree: Dict[str, Dict[str, Union[str, List]]]) -> Dict:
```

Comment:
We should probably have some sort of standalone unit tests for this function, similar to `day_night_map` even if the bulk of the testing is effectively done within `test_ModalFromGroupings.py`.

Reply:
Added.